content/projects/data-science-specific/data-wrangling/_index.md

```diff
@@ -65,29 +67,39 @@ df.apply(get_percentage, axis = 1) #axis=1 applies the function to all columns
 
 ### Instructions
 
-This data contains [personality scores](personality_scores.csv) for learners, plus the [department](departments.csv) they applied for.
+1. Your repo should contain everything needed to replicate your work. It's good practice to structure your files well, so we'll expect you to have a separate directory for "data" and "notebook", so that your final file structure looks something like this: 
 
-1. Import the dataset [personality_scores.csv](personality_scores.csv). Examine the data frame for duplicates (based on ID), and drop any duplicates that exist. Use an assert statement to check that the new data frame is the length of the unique entries of the original data frame.
+```
+├──data
+│   ├──departments.csv
+│   └──personality_scores.csv 
+├──notebook
+│  └──data_wrangling.ipynb
+├──README.md
+├──requirements.txt
+└──.gitignore 
+```
+2. Import the dataset [personality_scores.csv](personality_scores.csv). This data should be named `personality_df` Examine the data frame for duplicates (based on ID), and drop any duplicates that exist. Use an assert statement to check that the new data frame is the length of the unique entries of the original data frame. This new dataframe should be named `personality_score`
```

Comment:
Let's also be consistent throughout the project about how we spell DataFrame
Since it is its own very specific kind of data structure I think it should be "DataFrame"